index.js

```diff
@@ -14,5 +14,81 @@ async function start() {
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
   // Now try and complete the program.
-  process.exit();
+
+  //Story One: The computer picks a number between 1 and 100 and guesses it against my secret number.
+
+  function randomNum(min, max) {
+    let range = max - min + 1
+
+
+    return Math.floor(Math.random() * range) + min
+  }
+
+  let firstGuess = randomNum(0, 100)
+  let guessCheck = await ask(`Is this your number? ${firstGuess} (Y)ES/(N)O? `)
+
+  //Story Two: If I respond with (Y)ES the game responds with a victory message.
+
+  if (guessCheck.toLowerCase() === 'yes' || guessCheck.toLowerCase() === 'y') {
+    console.log(`Your number was ${firstGuess}! Yay me!`)
+    process.exit()
+  }
+
+  //Story Three: If I respond with N(O) the game responds by asking if it's (H)IGHER or (L)OWER.
+
+  else if (guessCheck.toLowerCase() === 'no' || guessCheck.toLowerCase() === 'n') {
+    let hiLow = await ask('Is your number (H)IGHER or (L)OWER? ')
+
+    // Story Four: If I respond with (H)IGHER the range is modified with a new minimum.
+    // If I respond with (L)OWER the range is modified with a new maximum.
+    // Computer guesses a new value within that range, if incorrect it repeats the process.
+
```

Comment:
I saw your comment that Story Four is where you were stuck. You might consider having the condition for your `while` loop be `guessCheck === "no"`, and then have an `if () {//code} else if {//code}` inside of this `while` loop to handle if the guess needs to be higher or lower. Having higher and lower separated into their own `while` loops affects their relation to each other.